weblate/trans/util.py

```diff
@@ -171,7 +171,14 @@ def get_clean_env(extra=None):
     }
     if extra is not None:
         environ.update(extra)
-    variables = ('PATH', 'LD_LIBRARY_PATH', 'SystemRoot')
+    variables = (
+        'PATH',
+        'LD_LIBRARY_PATH',
+        'NSS_WRAPPER_GROUP',
+        'NSS_WRAPPER_PASSWD',
+        'LD_PRELOAD',
+        'SystemRoot',
+      )
```

Comment:
The list probably grew up too much, so it might be good to document why each variable is needed.

Reply:
OK, added a comment. Do you find it readable enough?